### PR TITLE
Kevinf sprint9

### PR DESCRIPTION
**Pull request by kfansler**
Changes:
* OAuth section modified to include optional plain text (as opposed to using a hash). Not yet implemented for SDKs, so just the table and the cURL examples include the new simplified method.
* Query param for GET /sheets for ?rowsModifiedSince

### CODE REVIEW

File: source/_introduction.html.md.erb [hunk 1]
```diff
@@ -1116,7 +1128,9 @@ smartsheet.tokens.refreshAccessToken(options)
 ```
 
 ```csharp
-// [todo]
+Token tokenRefreshed = oauth.RefreshToken(
+  token           // Token token
```
Comment (abryan-smartsheet) [inline source/_introduction.html.md.erb]: This comment doesn't really add anything helpful, IMO. Can this be condensed into:
Token tokenRefreshed = oauth.RefreshToken(token);

File: source/_introduction.html.md.erb [hunk 2]
```diff
@@ -1116,7 +1128,9 @@ smartsheet.tokens.refreshAccessToken(options)
...
+);
 ```
 
 ```java
```
Comment (abryan-smartsheet) [inline source/_introduction.html.md.erb:1174]: Token refreshToken = oAuthFlow.refreshToken(token);

File: source/_introduction.html.md.erb (hunk 1 above)
Comment (kfansler) [reply source/_introduction.html.md.erb]: Agreed.

File: source/_enterprise.html.md.erb [hunk 3]
```diff
@@ -4,6 +4,21 @@ The following section describes the specific event types and payloads returned f
 
 With Event Reporting, you can programmatically retrieve events that are occurring in your Smartsheet organization account. Examples of events are create, update, load, and delete of sheets, reports, dashboards, attachments, and users. Note: while the current list of events covers most of Smartsheet functionality, new events will be added to the list as Smartsheet introduces new functionality.
 
+Some actions in Smartsheet are performed by system components. Therefore, the userId property in some response payloads refers to a system userId and not to a regular user. For example, the Smartsheet Events API shows 5629504736520068 as the userId when an anonymous user accesses a sheet that is published for anyone.
```
Comment (PauloLisboa) [inline source/_enterprise.html.md.erb]: Minor suggestion (just changed from`system userId` to `system user`):
the userId property in some response payloads refers to a system **user** and not to a regular user

Comment (kfansler) [reply source/_enterprise.html.md.erb]: Thanks!

**Pull request merged**